rclcpp/include/rclcpp/utilities.hpp

```diff
@@ -210,6 +210,12 @@ sub_will_underflow(const T x, const T y)
   return (y > 0) && (x < (std::numeric_limits<T>::min() + y));
 }
 
+const char *
+get_c_string(const char * string_in);
+
+const char *
```

Comment:
I think you just need `RCLCPP_PUBLIC` on each of these functions.

Reply:
Added `RCLCPP_PUBLIC` to `get_c_string` functions. I didn't know that it was necessary for Windows. I use only Linux :S I hope it works with this changes.

I also documented both functions and macros.